aws/network-primitives/dns.tf

```diff
@@ -20,13 +20,38 @@ resource "cloudflare_record" "rpc" {
 # }
 
 resource "cloudflare_record" "core-domain" {
-  count   = length(local.domain_node_ip_v4)
+  count   = length(local.domain_nodes_ip_v4)
   zone_id = data.cloudflare_zone.cloudflare_zone.id
   name    = "${var.domain-node-config.domain-prefix}.${var.network_name}"
-  value   = local.domain_node_ip_v4[count.index]
+  value   = local.domain_nodes_ip_v4[count.index]
   type    = "A"
 }
 
+resource "cloudflare_record" "core-domain_ipv6" {
+  count   = length(local.domain_nodes_ip_v4)
```

Comment:
If you see the count using ipv4 value and ipv6 for the dns record value, this is not a mistake. AWS assigns and instance 2x IPv6 records for the interface, and if not done this way it will create extra AAAA records and bootstrap records. we will end up have `bootstrap-0..4` AAAA records otherwise.